session/src/main/scala/akka/cassandra/session/scaladsl/CassandraSession.scala

```diff
@@ -56,7 +55,7 @@ final class CassandraSession(
     extends NoSerializationVerificationNeeded {
 
   implicit private[akka] val ec = executionContext
-  private lazy implicit val materializer = ActorMaterializer()(system)
+  private implicit val sys: ActorSystem = system
```

Comment:
@jewertow could you revert this change to the `session` project? As @ennru said, we are anyway moving it to Alpakka and this would introduce conflicts with that work.
Otherwise this is ready to be merged.

Reply:
Done